README.md

```diff
@@ -403,46 +467,11 @@ Type: `config => transformedConfig`. Default `x => x` (identify function).
 If you want to make changes to the Webpack configuration beyond what's available in the above options, you can use this, the nuclear option.
 Your function receives the Webpack configuration that Underreact generates and returns a new Webpack configuration, representing your heart's desires.
 
-### jsEntry
-
-Type: `string`. Absolute path, please. Default: `${project-directory}/src/index.js`.
-
-The entry JS file for your app. Typically this is the file where you'll use `react-dom` to render your app on an element.
-
-In the default value, `project-directory` refers to the directory of your `underreact.config.js` file, or the current working directory.
-
-### outputDirectory
-
-Type `string`. Absolute path, please. Default: `${project-directory}/_underreact-site/`.
-
-The directory where files should be written.
-
-You'll want to ignore this directory with `.gitignore`, `.eslintignore`, etc.
-
-In the default value, `project-directory` refers to the directory of your `underreact.config.js` file, or the current working directory.
-
-### publicDirectory
-
-Type `string`. Absolute path, please. Default: `${project-directory}/src/public/`.
-
-Any files you put into this directory will be copied, without processing, into the [outputDirectory](#outputdirectory).
-You can put images, favicons, data files, anything else you want in here.
-
-In the default value, `project-directory` refers to the directory of your `underreact.config.js` file, or the current working directory.
-
-### publicAssetsPath
-
-Type: `string`. Default: `underreact-assets`.
-
-The directory where Underreact assets will be placed, relative to the site's root.
-
-By default, for example, the main JS chunk will be written to `underreact-assets/main.chunk.js`.
-
-It's important to know about this value so you can set up caching and other asset configuration on your server.
+### vendorModules
 
-### port
+Type: `Array<string>`. Default: `[]`.
```

Comment:
Should the default be `['react', 'react-dom']`?

Reply:
Yes it is, however I am a bit worried about the implementation as we concat this to an internal array of vendor. I wonder if that is of any real concern to user ?